<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Threshold</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Color</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Show</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="originalImageTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGraphicsView" name="originalImageView">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>751</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="grayImageTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGraphicsView" name="grayImageView">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>751</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="finalImageTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGraphicsView" name="finalImageView">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>751</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>319</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="thresholdSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addLayerButton">
       <property name="text">
        <string>Add layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="generateImageButton">
    <property name="geometry">
     <rect>
      <x>994</x>
      <y>280</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_image"/>
    <addaction name="actionSave_project"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionSave_image">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
